### PR TITLE
docs: Update custom extractor Poetry instructions

### PR DESCRIPTION
**Pull request by EMCP**
the build and publish command have been changed in poetry

### CODE REVIEW

**Comment by EMCP** [comment]
@afolson , thanks for the quick response.. hope this format works otherwise I can use the multi line ```bash approach

```bash
like
so
```

**Review by afolson — APPROVED** [review]

**Comment by afolson** [comment]
@edgarrmondragon Is this the best way to do this now? Looks like `--build` was deprecated a bit ago.

**Review by edgarrmondragon — APPROVED** [review]

File: docs/src/_tutorials/custom-extractor.md [hunk 1]
```diff
@@ -470,8 +470,8 @@ your tap directly to PyPI.
 
 1. Create an account with [PyPI](https://pypi.org).
 2. Create a PyPI API token for use in automated publishing. (Optional but recommended.)
-3. Run `poetry --build publish` from within your repo to build and push your latest version
-   to the PyPI servers.
+3. Run `poetry build` from within your repo to build
```
Comment (edgarrmondragon) [inline docs/src/_tutorials/custom-extractor.md:473]:
```suggestion
3. Run `poetry build` from within your repo to build.
```

**Review by edgarrmondragon — CHANGES_REQUESTED** [review]

**Comment by edgarrmondragon** [comment]
> @edgarrmondragon Is this the best way to do this now? Looks like `--build` was deprecated a bit ago.

@afolson yup, the official docs now use commands `build` and `publish`

**Review by tayloramurphy — APPROVED** [review]

**Review by edgarrmondragon — APPROVED** [review]

**Comment by edgarrmondragon** [comment]
Thanks @EMCP!

**Pull request merged**